examples/TestAppUwp/MainPage.xaml.cs

```diff
@@ -262,47 +75,54 @@ public static string GetDeviceName()
 
         public MainPage()
         {
+            RestoreSettings();
+
             this.InitializeComponent();
 
-            muteLocalVideoStroke.Visibility = Visibility.Collapsed;
-            muteLocalAudioStroke.Visibility = Visibility.Collapsed;
+            // Insert items manually into navigation menu. It doesn't seem that there is a way
+            // to force selecting a menu item when using data binding, so selecting the first
+            // page below would crash (empty MenuItems list if assigning MenuItemsSource).
+            //foreach (var catBase in Categories)
+            //{
+            //    if (catBase is Category cat)
+            //    {
+            //        var item = new NavigationViewItem()
+            //        {
+            //            Content = cat.Name,
+            //            Icon = new SymbolIcon() { Symbol = cat.Glyph },
+            //            DataContext = catBase
+            //        };
+            //        AutomationProperties.SetName(item, cat.Name);
+            //        navigationView.MenuItems.Add(item);
+            //    }
+            //    else if (catBase is Header headerCat)
+            //    {
+            //        var item = new NavigationViewItemHeader()
+            //        {
+            //            Name = headerCat.Name,
+            //            DataContext = catBase
+            //        };
+            //        AutomationProperties.SetName(item, headerCat.Name);
+            //        navigationView.MenuItems.Add(item);
+            //    }
+            //    else if (catBase is Separator sepCat)
+            //    {
+            //        navigationView.MenuItems.Add(new NavigationViewItemSeparator());
+            //    }
+            //}
```

Comment:
Delete?

Reply:
I need to fix a bug in navigation, need to keep that code around until I understand how to fix the bug, as this may be needed.